src/components/Table.js

```diff
@@ -90,6 +90,10 @@ type TableProps = {
    * be re-rendered with new values for the sortColumn and rows properties.
    */
   onSort: null | (sortColumn: number) => void;
+  /**
+   * The boolean flag if set, the selection of the row will be retained
```

Comment:
We might want to put the full descriptive comment here and have the comment on the property in the MasterDetails component say something like "See the noEmptySelection property of the Table component for details."